testsuite/run_forge.sh

```diff
@@ -242,15 +242,21 @@ elif [ "$FORGE_RUNNER_MODE" = "k8s" ]; then
     kubectl wait -n default --timeout=5m --for=condition=Ready "pod/${FORGE_POD_NAME}"
 
     # tail the logs and tee them for further parsing
+    echo "=====START FORGE LOGS====="
     kubectl logs -n default -f $FORGE_POD_NAME | tee $FORGE_OUTPUT
+    echo "=====END FORGE COMMENT====="
+
+    # wait for the pod status to change potentially
+    sleep 10
```

Comment:
You don't need this ?

Reply:
doesn't hurt to spin for another 10 seconds, but just to be safe